### DISCUSSION TITLE
Is there a way to make LaunchMenu resizable?

### DISCUSSION THREAD

**Discussion by faressoft** [LaunchMenu/LaunchMenu#188] [General] [answered]
Is there a way to make LaunchMenu resizable without the need to change the window size from the window manager?

**Comment by faressoft** [comment, marked as answer]
remote.getCurrentWindow().setResizable(true);

**Comment by TarVK** [comment]
Yes, this does not even have to be done programmatically!

There's a setting you can find to enter a new window size precisely, but while the setting is selected you can also drag the window as you requested:
![image](https://github.com/LaunchMenu/LaunchMenu/assets/26687938/ce1ff8cf-7def-46f4-a451-c25f6b40350b)

We chose to not have the window drag resizing enabled by default, for stylistic reasons. This behavior comes from the [window-manager](https://github.com/LaunchMenu/LaunchMenu/tree/development/packages/core-applets/window-manager) applet however, and could be modified.
